.circleci/config.yml

```diff
@@ -433,10 +429,6 @@ jobs:
 
       # Setup Angular ivy snapshots built with ngtsc.
       - run: node ./scripts/circleci/setup-angular-snapshots.js --tag master-ivy-aot
-      # Disable type checking when building with Ivy. This is necessary because
-      # type checking is not complete yet and can incorrectly break compilation.
-      # Issue is tracked with FW-1004.
-      - run: sed -i "s/\(_ENABLE_NG_TYPE_CHECKING = \)True/\1False/g" tools/defaults.bzl
```

Comment:
Can you remove the `ENABLE_NG_TYPE_CHECKING` flag from `defaults.bzl` too? We just had that so that we can disable type checking here?